<launch>                                                                                             
    <arg name="file_path" default="/home/amsl/catkin_ws/bag_data/" />
    <arg name="file_name" default="semantickitti_sequence00" />
    <node name="bag_data" pkg="rosbag" type="play" args="-l $(arg file_path)$(arg file_name).bag">   
        <remap from="/semantickitti/camera_color_right/image_raw" to="/image_view/output" />
    </node>

    <!-- Launch ROS Node for net handler with all the desired parameters-->
    <node pkg="semantic_mesh_localization" type="bonnet_node" name="bonnet_node" output="screen">
        <!-- Topics -->
        <rosparam command="load" file="$(find bonnet_run)/config/topic_cfg.yaml" />
        <!-- CNN stuff -->
        <rosparam command="load" file="$(find bonnet_run)/config/cnn_cfg.yaml" />
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find semantic_mesh_localization)/config/test_image_segmentation.rviz" />

</launch>                          
